src/TranslationLoaderManager.php

```diff
@@ -20,7 +23,15 @@ public function load($locale, $group, $namespace = null): array
     {
         $fileTranslations = parent::load($locale, $group, $namespace);
 
-        if (! is_null($namespace) && $namespace !== '*') {
+        $model = config('translation-loader.model');
+        $model = (new $model);
+        try {
+            DB::connection()->getPdo();
+        } catch (\Exception $e) {
+            return $fileTranslations;
+        }
+
+        if (! is_null($namespace) && $namespace !== '*' || ($model instanceof Model && !Schema::hasTable($model->getTable()))) {
```

Comment:
This condition is too complex. Could you extract this to a function that describes what is being checked? If possible, use early returns in that new function.